book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    Rental.new(date, self, person)
```

Comment:
- [ ] Kindly use the arrangement of the Rental constructor parameters (book, person, _date).

  ```
  Rental.new(self, person, date)
  ```

Reply:
resolve